tests/test_h5store.py

```diff
@@ -406,15 +405,12 @@ class Foo(object):
             self.assertEqual(h5s[key], d)
 
     def test_keys_with_dots(self):
-        with warnings.catch_warnings(record=True) as w:
-            warnings.simplefilter('always')
+        with self.assertRaises(InvalidKeyError):
```

Comment:
Same as above - I'd move this around and wrap each of the lines that are expected to trigger failure in the `assertRaises` block.

Reply:
See comment above.